src/webgpu/util/check_contents.ts

```diff
@@ -263,10 +263,10 @@ ${generatePrettyTable(opts, [
 // Helper helpers
 
 /** Convert an integral `number` into a hex string, padded to the specified `byteLength`. */
-function intToPaddedHex(number: number, { byteLength }: { byteLength: number }) {
-  assert(Number.isInteger(number), 'number must be integer');
-  let s = Math.abs(number).toString(16);
+function intToPaddedHex(n: number | bigint, { byteLength }: { byteLength: number }) {
+  assert(Number.isInteger(n), 'number must be integer');
+  let s = (n < 0n ? -n : n).toString(16);
```

Comment:
nit: maybe rename `n` such that it doesn't also match the suffix for the BigInt value of `0n`.